convert_proto.sh

```diff
@@ -1,6 +1,6 @@
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/..
 
 # NEEDS TO BE SET MANUALLY
-ENV_DIR=$CONDA_PREFIX
+TWO_TO_THREE_PATH=/home/taguhi/miniconda3/envs/pybmix_foo/bin/2to3
```

Comment:
No I mean that `2to3` should be in the `PATH` environmental variable. So that typing `2to3` in the command line will run it.

Reply:
We misunderstood first. Now it is assumed to be in the environment variable
$PYTHONPATH and in the convert_proto.sh 2to3 is called directly.